gui/sessionselector.h

```diff
@@ -0,0 +1,61 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef SESSION_SELECTOR_DIALOG_H
+#define SESSION_SELECTOR_DIALOG_H
+
+#include "gui/dialog.h"
+#include "gui/widgets/list.h"
+#include "common/fs.h"
+#include "common/hashmap.h"
+#include "common/stack.h"
+#include "common/str.h"
+
+#include "scumm/he/intern_he.h"
```

Comment:
Oops. You cannot do it from the GUI. This means that you have to move these two files, `sessionselector.*` back to `engines/scumm`. Perhaps, call it there `dialog_sessionselector.h`